lambda/translmod.ml

```diff
@@ -254,37 +259,44 @@ let init_shape id modl =
               const_int 1 (* camlinternalMod.Lazy *)
           | _ ->
               let not_a_function =
-                Unsafe {reason=Unsafe_non_function; loc; subid }
+                Unsafe {reason=Unsafe_non_function; loc; subid; path }
```

Comment:
Here the path to the unsafe item is `Pdot(path,subid)`: making the meaning of the `path` field consistent will remove the need of keeping `subid` on the side to reconstruct the name in `get_name`.